akka-http2-support/src/main/scala/akka/http/impl/engine/http2/BufferedOutletSupport.scala

```diff
@@ -75,18 +75,27 @@ trait BufferedOutletSupport { logic: GraphStageLogic ⇒
       elem.trigger()
     }
 
+    /** Buffer an element to be used as initial element to be pushed once a second element arrives. */
+    final def buffer(element: T): Unit =
```

Comment:
these are just random ideas so far... Instead I'll likely manually push the Settings in the right places.